examples/cms-sitecore/components/alert.js

```diff
@@ -0,0 +1,38 @@
+import Container from './container'
+import { EXAMPLE_PATH } from '../lib/constants'
+import Link from 'next/link'
+import cn from 'classnames'
+
+export default function Alert({ preview }) {
+  return (
+    <div
+      className={cn('border-b', {
+        'bg-accent-7 border-accent-7 text-white': preview,
+        'bg-accent-1 border-accent-2': !preview,
+      })}
+    >
+      <Container>
+        <div className="py-2 text-center text-sm">
+          {preview ? (
+            <>
+              This is page is a preview.{' '}
+              <Link href="/api/exit-preview" className="underline hover:text-cyan duration-200 transition-colors">
```

Comment:
We can use a regular `a` linking to an API route 👍

Reply:
For some reason, an existing ES Lint rule prevent the use the regular a tag. 
Giving the error below during build :
Error: Do not use the HTML <a> tag to navigate to /api/exit-preview/. Use Link from 'next/link' instead. See: https://nextjs.org/docs/messages/no-html-link-for-pages.  @next/next/no-html-link-for-pages
info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules

Reply:
Got it (cc @housseindjirdeh )

We can leave it as is for now.